src/towncrier/_project.py

```diff
@@ -39,15 +44,31 @@ def _get_package(package_dir: str, package: str) -> ModuleType:
     return module
 
 
-def get_version(package_dir: str, package: str) -> str:
-    module = _get_package(package_dir, package)
+def _get_metadata_version(package: str) -> str | None:
+    distributions = packages_distributions()
+    distribution_names = distributions.get(package)
```

Comment:
if `distribution_names` is `None` then the `len()` will raise a `TypeError`. Maybe raise a better error in this case?

Reply:
It's now `not distribution_names or len(distribution_names) != 1`, returning `None`. That seems fit for purpose...